### PR TITLE
Add skidl and kinparse

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/skidl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e0853614f8271ae429e715cfb7a1e505af40d1e8caee1b5cf08c6d269e69e631
+
+build:
+  number: 0
```
Comment (marcelotrevisani) [inline recipes/skidl/meta.yaml]: Do you know if this recipe cannot be noarch for the same reason as the another one?

Comment (hmaarrfk) [reply recipes/skidl/meta.yaml]: When I did, the windows build complained that there was an exe

Comment (scopatz) [reply recipes/skidl/meta.yaml]: only linux builds matter for noarch

Comment (hmaarrfk) [reply recipes/skidl/meta.yaml]: I see. Will fix

**Comment by hmaarrfk** [comment]
@marcelotrevisani I think this is ready for a second review

**Comment by scopatz** [comment]
Thanks @hmaarrfk!

**Pull request merged**